app/uk/gov/hmrc/individualsemploymentsapi/sandbox/v1/SandboxData.scala

```diff
@@ -16,13 +16,13 @@
 
 package uk.gov.hmrc.individualsemploymentsapi.sandbox.v1
```

Comment:
Do we still need sandbox data?

Reply:
We need it for V1, only V2 is being removed.

Reply:
👍

Reply:
Yes. Its employments so has a V1 :-)